### DISCUSSION TITLE
About gatsby-plugin-wordpress

### DISCUSSION THREAD

**Discussion by jenishshrestha** [gatsbyjs/gatsby#29700] [Help]
I am new to Gatsby and I was about to try implementing gatsby-plugin-wordpress for automatic page creation but this plugin seems to be removed from github.

https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-wordpress

Does anyone know if this is still possible?

**Comment by LekoArts** [comment]
https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-wordpress is the right link :)

**Reply by jenishshrestha** [reply]
They are both same?
So, https://www.gatsbyjs.com/plugins/gatsby-plugin-wordpress/ and https://www.gatsbyjs.com/plugins/gatsby-source-wordpress/ are same?

**Reply by LekoArts** [reply]
They are not the same. You should use `gatsby-source-wordpress` and implement it yourself I think. I'll tag @TylerBarnes about this to remove this confusion. Here's the repo btw: https://github.com/TylerBarnes/gatsby-plugin-wordpress

**Reply by jenishshrestha** [reply]
Thank you :)

**Reply by TylerBarnes** [reply]
@jenishshrestha gatsby-plugin-wordpress with automatic page creation was an experiment that ended up being the wrong direction - sorry for the confusion here. I've deprecated that package just now to prevent this in the future. 

For automatic page creation check out the new [Filesystem Route API](https://www.gatsbyjs.com/docs/reference/routing/file-system-route-api/).

**Reply by jenishshrestha** [reply]
Thank you for guiding me to right path. I will look into this api :)
